middlewares/middlewares_utils.go

```diff
@@ -1,4 +1,4 @@
-package cmd
+package middlewares
```

Comment:
I don't think it's necessary to put the package name as a prefix `middlewares_utils`, used `utils` example. The name utils is not very good, we need to think of another name.